dandi/tests/fixtures.py

```diff
@@ -521,7 +521,7 @@ def _create_nwb_files(video_list):
             name=f"MouseWhiskers{no}",
             format="external",
             external_file=[str(vid_1), str(vid_2)],
-            starting_frame=[0],
+            starting_frame=[0, 0],
```

Comment:
```suggestion
            starting_frame=[0, 10],
```

Reply:
Where did you get 10?  To me, [it looks like](https://github.com/dandi/dandi-cli/blob/7c42ed91c084e978f9904fd7495f8fcf97239b1a/dandi/tests/fixtures.py#L467-L486) there are two frames per video.